mobile/Makefile

```
@@ -0,0 +1,173 @@
+## Default
+## $ make
+## sets up system for
+## $ tns run (android|ios)
+
+.DEFAULT_GOAL := apply-patches
+
+##
+## Release related targets
+##
+
+gradle-simul:
+	@( cd app/App_Resources/Android; cp app.gradle.simul app.gradle )
+
+gradle-prod:
+	@( cd app/App_Resources/Android; cp app.gradle.prod app.gradle )
+
+clean: gradle-simul
+	rm -rf node_modules platforms hooks
+
+npm-install: clean
+	npm ci
+	find app -name "*js" | xargs rm -f
+
+apply-patches: npm-install
+	patch -p0 < nodeify_temporary_patch.patch
+	cp nodeify_noxml.js node_modules/nativescript-nodeify/nodeify.js
+	rm -rf node_modules/public-encrypt/test/
+	if [ ! -d app/lib/dynacred ]; then \
+	  npm link ../dynacred; \
+	fi
+
+android-dev: apply-patches
+	tns prepare android
+
+roster-check:
+	   @if [ ! "$$PERSONHOOD3_ANDROID_PASS" ]; then echo "Please set PERSONHOOD3_ANDROID_PASS"; exit 1; fi
```

I find it more readable as

```suggestion
	   [ -n "$$PERSONHOOD3_ANDROID_PASS" ]
```